### ISSUE TITLE
Minor UI: Long names make cards longer...

### ISSUE THREAD

**Issue by johnmark13** [vessenes/persona#21]
![image](https://github.com/user-attachments/assets/750f9168-289f-4dcf-9258-39766e34bb89)

The stats get chopped off and probably bids/asks too...

**Comment by vessenes** [comment]
Yeah, this is tough. We give people 64 characters for names but we'll lose the grid layout if we expand a particular Persona Card. I'm open to suggestions. Maybe we could make it animate? Or just clip it and show on hover?

**Comment by johnmark13** [comment]
I guess, thinking about the primary use case of the market would lead me towards truncating the name and rendering in the overlay - the name being secondary... but, I guess the same cards are used on the Collective pages, and there we might think that our names are a prize part in the curation of the cabinet of curiosities! So either option on dealing with the text, but 100% keep the cards/font/layout itself fixed or it'll be messy.
